### PR TITLE
Proposal to structure the content + some minor edits

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

**Review by mcr — COMMENTED** [review]
I'm not sure from reading the diff what the goal is here.

**Comment by boucadair** [comment]
* have a better flow of the narrative text. For example, introduce first the meaning of each column before talking about ranges, etc.
* Ease identifying each IANA action
* Associate the DE guidance with the sub registry, not the registry group.
* avoid redundancy by pointing to the specific 8126 sections
* avoid embedded URLs, but use refs
* add pointers for some specific items (even if these are well-known)

File: draft-ietf-opsawg-pcaplinktype.md [hunk 1]
```diff
@@ -33,11 +33,12 @@ venue:
 
 normative:
 informative:
- Radiotap:
+  Radiotap:
```
[the remark this answers is not in the document]
Comment (guyharris) [reply draft-ietf-opsawg-pcaplinktype.md:36]: It appears the extra space before "Radiotap:" was removed.

[the remark this answers is not in the document]
Comment (boucadair) [reply draft-ietf-opsawg-pcaplinktype.md:36]: Yes. Thanks.

**Comment by boucadair** [comment]
> We're tryin to avoid that problem with the pcap/pcapng link-layer type registry, so many of the entries in the table will need one or more references to make it clear what format the packet data is in (and to keep the description in the table short!).

Thanks. Sounds good to me.

> So would the document reference in the Reference column:
> 
> > Reference: Indicates an authoritative the document reference for the LinkType or a requester reference.
> 
> be to such a document? Or, perhaps, to one or more such documents?

ACK. One r more documents can be listed there

**Pull request merged**